tests/test_acasclient.py

```diff
@@ -2296,6 +2296,144 @@ def test046_cmpdreg_admin_crud(self):
         self.client.delete_cmpdreg_vendor(vendor['id'])
         
 
+    def test_047_load_sdf_with_salts(self):
+        """
+        Test to Load SDF File w/ Salts to Determine Process
+        """
+        test_047_load_sdf_with_salts_file = Path(__file__).resolve().parent.\
+            joinpath('test_acasclient', 'test_047_register_sdf_with_salts.sdf')
+        mappings = [
+            {
+                "dbProperty": "Parent Corp Name",
+                "defaultVal": None,
+                "required": False,
+                "sdfProperty": "Corporate ID"
+            },
+            {
+                "dbProperty": "Lot Chemist",
+                "defaultVal": "bob",
+                "required": True,
+                "sdfProperty": "Lot Scientist"
+            },
+            {
+                "dbProperty": "Project",
+                "defaultVal": "PROJ-00000001",
+                "required": True,
+                "sdfProperty": "Project Code Name"
+            },
+            {
+                "dbProperty": "Parent Stereo Category",
+                "defaultVal": "unknown",
+                "required": True,
+                "sdfProperty": None
+            },
+            {
+                "dbProperty": "Lot Salt Abbrev",
+                "defaultVal": None,
+                "required": False,
+                "sdfProperty": "Lot Salt Name"
+            },
+            {
+                "dbProperty": "Lot Salt Equivalents",
+                "defaultVal": None,
+                "required": False,
+                "sdfProperty": "Lot Salt Equivalents"
+            }
+        ]
+
```

Comment:
nit: This could do with a code comment indicating that you're ensuring `HCl` is registered as a salt since it's needed for your test.